src/js/services/feeService.js

```diff
@@ -15,45 +15,45 @@ angular.module('copayApp.services').factory('feeService', function($log, $stateP
     return configService.getSync().wallet.settings.feeLevel || 'normal';
   };
 
-  root.getCurrentFeeValue = function(wallet, cb) {
+  root.getCurrentFeeValue = function(network, cb) {
+    network = network || 'livenet';
     var feeLevel = root.getCurrentFeeLevel();
 
-    wallet.getFeeLevels(wallet.credentials.network, function(err, levels) {
-      if (err)
-        return cb({
-          message: 'Could not get dynamic fee'
-        });
+    root.getFeeLevels(function(err, levels) {
+      if (err) return cb(err);
 
-      var feeLevelValue = lodash.find(levels, {
+      var feeLevelValue = lodash.find(levels[network], {
         level: feeLevel
       });
-      if (!feeLevelValue || feeLevelValue.feePerKB == null)
+
+      if (!feeLevelValue || !feeLevelValue.feePerKB)
         return cb({
-          message: 'Could not get dynamic fee for level: ' + feeLevel
+          message: gettextCatalog.getString('Could not get dynamic fee for level: ') + feeLevel
```

Comment:
It is better to include the argument inside the string so translators can do a better job.